runtime/src/lib.rs

```diff
@@ -358,6 +361,18 @@ pub fn handle_mailbox_commands(drivers: &mut Drivers) -> ! {
     // Indicator to SOC that RT firmware is ready
     drivers.soc_ifc.assert_ready_for_runtime();
     caliptra_drivers::report_boot_status(RtBootStatus::RtReadyForCommands.into());
+    // Disable attestation if in the middle of executing an mbox cmd during warm reset
+    if drivers.mbox.cmd_busy() {
+        let reset_reason = drivers.soc_ifc.reset_reason();
+        if reset_reason == ResetReason::WarmReset {
+            let mut result = DisableAttestationCmd::execute(drivers);
+            if let Ok(mut resp) = result {
+                Packet::copy_to_mbox(drivers, &mut resp);
```

Comment:
I don't think we need to copy the response to the mailbox since we're just executing it ourselves.

But once we call disable attestation, we probably want to get the mailbox back into idle state so that other mailbox commands still work.

Reply:
Done